### PR TITLE
update rebuilder-debian.sh for current debrebuild from devscripts

### PR DESCRIPTION
**Pull request by h01ger**
happy to rework if needed..!

### CODE REVIEW

File: worker/rebuilder-debian.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 set -xe
-# debrebuild.py needs to be run from the repo
-/debrebuild/debrebuild.py --output="${REBUILDERD_OUTDIR}" --builder=mmdebstrap --use-metasnap -- "${1}"
+cd "$(dirname "$1")"
+# for production its useful to call debrebuild with --cache=$directory
```
Comment (kpcyrd) [inline worker/rebuilder-debian.sh:4]:
```suggestion
# for production it's useful to call debrebuild with --cache="$directory"
```
besides that LGTM <3

**Comment by h01ger** [comment]
thanks, fix --force pushed

**Comment by h01ger** [comment]
uhm

**Comment by h01ger** [comment]
now..

**Review by kpcyrd — APPROVED** [review]

**Comment by kpcyrd** [comment]
Thanks! :sparkles:

**Pull request merged**